src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/WorldCustomSound.java

```diff
@@ -23,7 +23,7 @@ public WorldCustomSound(ConnectionImpl connection) {
 
 	public static ClientBoundPacketData create(ProtocolVersion version, String id, int x, int y, int z, float volume, float pitch) {
 		ClientBoundPacketData serializer = ClientBoundPacketData.create(PEPacketIDs.PLAY_SOUND);
-		StringSerializer.writeString(serializer, version, id);
+		StringSerializer.writeString(serializer, version, id.replaceAll("minecraft:", ""));
```

Comment:
Then use namespacedkeyutils to read the key. Still will be a lot faster than a regex.